### PR TITLE
Add database checking

### PR DESCRIPTION
**Pull request by PxyUp**

### CODE REVIEW

File: apps/internal/job/BUILD.bazel [hunk 1]
```diff
@@ -36,9 +45,16 @@ go_test(
         "job_grpc_test.go",
         "job_tcp_test.go",
         "job_sitemap_test.go",
+         "job_mongo_test.go",
```
Comment (PxyUp) [inline apps/internal/job/BUILD.bazel]: Formating

File: apps/internal/job/job_postgres.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			Location: m.location,
+			Port:     m.port,
+		},
+	}
+}
+
+func (j *postgresJob) Do() CheckError {
+	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
+		j.host, j.port, j.user, j.password, j.dbname)
+	db, err := j.postgresOpen("postgres", psqlInfo)
```
Comment (PxyUp) [inline apps/internal/job/job_postgres.go:78]: You are sure about the first argument?

Comment (DreamAndDrum) [reply apps/internal/job/job_postgres.go:78]: yes

File: apps/internal/job/job_mongo.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+		location:    location,
+	}
+}
+
+func (m *mongoError) GetLogData() *clientPb.Log {
+	return &clientPb.Log{
+		Code:        m.code,
+		Description: m.description,
+		Meta: &clientPb.MetaData{
+			Id:       uuid.New().String(),
```
Comment (PxyUp) [inline apps/internal/job/job_mongo.go:59]: In all files in meta data you lost many argument

File: apps/internal/job/job_mongo.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+	clientOptions := options.Client().ApplyURI(j.url)
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+	client, err := j.mongo.connect(ctx, clientOptions)
+	if err != nil {
+		return newMongoError(logId, startTime, ptypes.TimestampNow(), clientPb.StatusCode_Error, mongoConnectionError.Error(), j.url)
+	}
+
+	fmt.Println(client)
```
Comment (PxyUp) [inline apps/internal/job/job_mongo.go:77]: Some fmt.Println and timeout options should be configurated out side

**Review by DreamAndDrum — APPROVED** [review]

**Pull request closed without merging**